README.md

```diff
@@ -92,7 +92,8 @@ for (pid, process) in sys.get_processes() {
 
 // Display system information:
 println!("System name:      {:?}", sys.get_name());
-println!("System version:   {:?}", sys.get_version());
+println!("System kernel version:   {:?}", sys.get_kernel_version());
```

Comment:
Please keep the `{:?}` parts aligned between themselves. (Yup, little format freak here haha)

Reply:
fixed in 1aff6db